pkg/cnab/config_adapter/adapter.go

```diff
@@ -9,25 +9,28 @@ import (
 	"github.com/deislabs/cnab-go/bundle/definition"
 	"github.com/deislabs/porter/pkg/cnab/extensions"
 	"github.com/deislabs/porter/pkg/config"
+	"github.com/deislabs/porter/pkg/context"
+	"github.com/deislabs/porter/pkg/manifest"
 )
 
 const SchemaVersion = "v1.0.0"
 
 // ManifestConverter converts from a porter manifest to a CNAB bundle definition.
 type ManifestConverter struct {
-	*config.Config
+	*context.Context
+	Manifest     *manifest.Manifest
 	ImageDigests map[string]string
 }
 
-func NewManifestConverter(cfg *config.Config, imageDigests map[string]string) *ManifestConverter {
+func NewManifestConverter(cxt *context.Context, manifest *manifest.Manifest, imageDigests map[string]string) *ManifestConverter {
 	return &ManifestConverter{
-		Config:       cfg,
+		Context:      cxt,
+		Manifest:     manifest,
 		ImageDigests: imageDigests,
 	}
 }
 
 func (c *ManifestConverter) ToBundle() *bundle.Bundle {
-	fmt.Fprintf(c.Out, "\nGenerating Bundle File with Invocation Image %s =======> \n", c.Manifest.Image)
```

Comment:
I decided to remove these output lines, as doing a conversion from one data structure to another doesn't really warrant these log lines anymore. They maybe made sense last November for the prototype but now aren't necessary and are just noise.